play_by_play.py

```diff
@@ -1,48 +1,28 @@
 import pandas as pd
 import numpy as np
-import time
 import config
 from connect import Connector
-from nba_api.stats.endpoints import playbyplayv2
 from games import Games
 
 
 class PlayByPlay:
 
     columns = config.play_by_play_columns
 
-    def __init__(self, game_ids):
-        self.game_ids = game_ids
+    def __init__(self, raw_plays_dfs, max_play_id):
+        self.max_play_id = max_play_id
+        self.raw_plays_dfs = raw_plays_dfs
 
     def get_plays(self):
         return self._clean_data()
 
     def _build_raw_dataframe(self):
```

Comment:
What is the purpose of this function? Please add a comment for it.

Reply:
Which function?